rita_client/src/exit_manager/mod.rs

```diff
@@ -590,11 +591,62 @@ fn initialize_selected_exit_list(exit: String, server: ExitServer) {
     });
 }
 
+/// Retrieve a unix timestamp from the exit's mesh IPv6
```

Comment:
this comment should explain that this is used to sync the clients clock since routers have no internal clock.